go-api-client/src/go-client-generated/docs/Run.md

```diff
@@ -21,8 +21,7 @@ Name | Type | Description | Notes
 **TimeO2Start** | **int64** |  | [optional] [default to null]
 **TimeTrgEnd** | **int64** |  | [optional] [default to null]
 **TimeTrgStart** | **int64** |  | [optional] [default to null]
-**TrgEnabled** | **bool** | Boolean if the trigger is enabled. | [optional] [default to null]
```

Comment:
Default should be `OFF`

Reply:
This is within the go-api so I think no changes are needed here